mood-analysis.rb

```diff
@@ -14,11 +18,44 @@ def analyze_mood(words)
       sad += 1
     end
   end
+
   return ":-)" if happy > sad
   return ":-(" if happy < sad
   return ":-|"
 end
 
+def happy_days(result)
+  happy_days = 0
+  day_count = 0
+
+  result.each do |journal|
+
+    mood = journal[1]
+
+    if happy_days < 3
+      day_count += 1
+       happy_days += 1 if mood == ":-)"
+    end
+  end
+  day_count
+end
+
+
+def overall_mood(result)
+
+  overall_mood_result = {}
+  overall_mood_result[":-)"] = 0
+  overall_mood_result[":-|"] = 0
+  overall_mood_result[":-("] = 0
+
+    result.each do |journal|
+      overall_mood_result[journal[1]] += 1
+    end
+
+    overall_mood_result.max_by {|mood, count| count}.first
+
+end
```

Comment:
If we don't do anything with journal[0] ever, why should we send `result` to overall_mood instead of just `text`?